### PR TITLE
Add Confluence Loader unit tests

### PR DESCRIPTION
**Pull request by adamjq**
Adds some basic unit tests for the ConfluenceLoader that can be extended later. Ports this [PR from llama-hub](https://github.com/emptycrown/llama-hub/pull/208) and adapts it to `langchain`.

@Jflick58 and @zywilliamli adding you here as potential reviewers

### CODE REVIEW

**Comment by adamjq** [comment]
Hi @hwchase17 this is ready for review again now

**Review by zywilliamli — APPROVED** [review]
thanks for adding the tests, looks good

**Comment by adamjq** [comment]
@hwchase17 could you please run the workflows for this?

File: pyproject.toml [hunk 1]
```diff
@@ -109,6 +109,10 @@ responses = "^0.22.0"
 pytest-asyncio = "^0.20.3"
 lark = "^1.1.5"
 pytest-mock  = "^3.10.0"
+atlassian-python-api = "^3.36.0"
```
Comment (dev2049) [inline pyproject.toml:112]: @eyurtsev so is right way to do this now to drop these from dependencies and just mark unit tests with pytest.mark.requires

**Pull request merged**